tests/queries/0_stateless/02184_default_table_engine.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE table_02184 (x UInt8); --{serverError 119}
+SET default_table_engine = 'Log';
+CREATE TABLE table_02184 (x UInt8);
+SHOW CREATE TABLE table_02184;
+DROP TABLE table_02184;
+SET default_table_engine = 'MergeTree';
+CREATE TABLE table_02184 (x UInt8); --{serverError 42}
+CREATE TABLE table_02184 (x UInt8, PRIMARY KEY (x));
+SHOW CREATE TABLE table_02184;
+DROP TABLE table_02184;
+CREATE TABLE test_optimize_exception (date Date) PARTITION BY toYYYYMM(date) ORDER BY date;
+SHOW CREATE TABLE test_optimize_exception;
+DROP TABLE test_optimize_exception;
+CREATE TABLE table_02184 (x UInt8) PARTITION BY x; --{serverError 36}
+CREATE TABLE table_02184 (x UInt8) ORDER BY x;
+SHOW CREATE TABLE table_02184;
+DROP TABLE table_02184;
+CREATE TABLE table_02184 (x UInt8) PRIMARY KEY x;
+SHOW CREATE TABLE table_02184;
+DROP TABLE table_02184;
+SET default_table_engine = 'Memory';
+CREATE TABLE numbers1 AS SELECT number FROM numbers(1000);
+SHOW CREATE TABLE numbers1;
+SELECT * FROM numbers1;
```

Comment:
Consider reducing number of rows from 1000 to 5-10 or querying some aggregates (like sum and count) instead of all rows